CHANGES.md

```diff
@@ -5976,15 +5971,15 @@ OpenSSL 1.0.1
    If the BIT STRING containing the signature has non zero unused bits reject
    the signature. All current signature algorithms require zero unused bits.
 
-   2. Check certificate algorithm consistency.
+   1. Check certificate algorithm consistency.
```

Comment:
This looks like a wrong fix. The numbering should be kept, probably the text needs to be reformatted to keep all the text as part of the list item.

Reply:
I restored the numbering.